New colors

Pap36 opened the pull request:
This PR adds a new color for `Success` Events (`Nordic Green`) and assigns two nordic colors to `md_theme_warning` and `md_theme_success` properties.

This PR also changes the `secondary` and `secondary_container` colors depending on the theme as follows:
`Night Theme`
  * `secondary` becomes `nordicLake` from `nordicBlue`
  * same for `secondary_container`
 
`Light Theme`
  * `secondary` becomes `nordicBlue` from `nordicLake`
  * same for `secondary_container`

Feel free to reject the latest commit if you don't like this.

Pap36 commented:
Using the original Colors:
`Light Theme`:
![image](https://github.com/NordicPlayground/Android-Common-Libraries/assets/61757553/1661256e-a62d-4f33-a4a7-8a82faf6704a)
![image](https://github.com/NordicPlayground/Android-Common-Libraries/assets/61757553/2a219d3d-0e51-4da1-8bf9-ad754aa45d7d)
`Dark Theme`:
![image](https://github.com/NordicPlayground/Android-Common-Libraries/assets/61757553/17025d3b-35fd-4d8c-89ce-398eeb47eff9) 
![image](https://github.com/NordicPlayground/Android-Common-Libraries/assets/61757553/76a586b6-b742-4891-a1d2-28d517c6e126)
Using the Proposed Colors:
`Light Theme`:
![image](https://github.com/NordicPlayground/Android-Common-Libraries/assets/61757553/2b1e98fc-826f-4a94-becc-8eb5bb0512b7)
![image](https://github.com/NordicPlayground/Android-Common-Libraries/assets/61757553/65c63f90-369d-4871-a9a7-72f18cfd7516)
`Dark Theme`:
![image](https://github.com/NordicPlayground/Android-Common-Libraries/assets/61757553/7b9c4bf9-49ef-436e-922d-eadd9a531ec5)
![image](https://github.com/NordicPlayground/Android-Common-Libraries/assets/61757553/8669e3b4-2187-4e93-a724-16278b073fe7)

I feel that using darker colors on the dark theme helps with consistency even if on the light theme the primary and secondary color are the same.

philips77 commented:
I prefer the original (before change), but perhaps because I got use to it..?
Why?
* FAB is more vibrant, not just boring primary color
* After proposed change all colors are the same on a screen, boring

Pap36 commented:
> I prefer the original (before change), but perhaps because I got use to it..? Why?
> 
> * FAB is more vibrant, not just boring primary color
> * After proposed change all colors are the same on a screen, boring

Yes but then in the light theme you have a darker FAB and menu selection whereas in the dark theme you have a lighter one which I think is weird.

Pap36 commented:
I rebased on new changes on main and reverted the secondary colors to the way they used to be. I also added a getter for `Nordic Green` color. I don't think I broke anything but just in case someone should review before accepting.
